files/SZWEEXEC/ZWEGEN00

```diff
@@ -454,58 +454,74 @@ end
 if FreeByDD('zwejcl') > 0 then do
   ExitWithRC(8)
 end
+
+clsQueue = ClearQueue()
 do j = 1 to !contentToRead.0
-  cmp1 = COMPARE(!contentToRead.j, 'CONFIG={ZWE_CLI_PARAMETER_CONFIG}')
-  cmp2 = COMPARE(!contentToRead.j, 'CONFIG=CFG.ZWE_CLI_PARAMETER_CONFIG')
-  if cmp1 = 0 | cmp2 = 0 then do
+  queue !contentToRead.j
+end
+
+do queued()
+  pull zoweslstcLine
+
+  cmp1 = COMPARE(zoweslstcLine, 'CONFIG={ZWE_CLI_PARAMETER_CONFIG}')
+  cmp2 = COMPARE(zoweslstcLine, 'CONFIG=CFG.ZWE_CLI_PARAMETER_CONFIG')
+  if 0 == cmp1 | 0 == cmp2 then do
     say 'Defining CONFIG entry for ZWESLSTC'
-    currentline = j
     if ReadFromDataSet('myconfig') = 0 then do
       do j = 1 to !contentToRead.0
-        type = WORD(!contentToRead.j, 1)
-        location = WORD(!contentToRead.j, 2)
-        element = type'('location')'
-        if j = 1 then do
-          cmd = 'LINE 'currentline' = "CONFIG='element
-          if !contentToRead.0 > 1 then do
-            cmd = cmd':\"'
+        parse var !contentToRead.j type location
+        shouldAddColon = !contentToRead.0 > j
+        element = type'('strip(location)')'
+        if 1 == j then do
+          element = 'CONFIG='element
+        end
+        if shouldAddColon then do
+          element = element':\'
+        end
+        eleLength = length(element)
+        MAX_LINE_LEN = 70 /* 70 allows PARMLIB to always fit in a single line */
```

Comment:
The max `PARMLIB` entry is in total 72 (if first entry):
```
----+----1----+----2----+----3----+----4----+----5----+----6----+----7--
CONFIG=PARMLIB(ZOWEHLQ.ZOWE.PR#4419.CONFIG00.YAML----.TEST-(ZOWEYAML)):\
```
Then it is split into this, which IMHO we should avoid
```
CONFIG=PARMLIB(ZOWEHLQ.ZOWE.PR#4419.CONFIG00.YAML----.TEST-(ZOWEYAML))\
:\
PARMLIB(ZOWEHLQ.ZOWE.PR#4419.YAML(YAML1)):\
PARMLIB(ZOWEHLQ.ZOWE.PR#4419.YAML(YAML2))
```

With instream data, we can go beyond 70 characters, the limit is record length 80.